docker/antora/return-code.py

```diff
@@ -3,6 +3,7 @@
 
 print('returning')
 def handler(event, context):
-    print('Running runAntoraAws.sh')
-    subprocess.run(["./runAntoraAws.sh"])
+    bucketname = event['Records'][0]['s3']['bucket']['name']
```

Comment:
may want to add some error handling here

Reply:
Just pushed an update to add a try, except on reading the bucket name